ledger-core/instrumentation/insapp/cloud_test.go

```diff
@@ -1,42 +0,0 @@
-// Copyright 2020 Insolar Network Ltd.
-// All rights reserved.
-// This material is licensed under the Insolar License version 1.0,
-// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
-
-package insapp_test
-
-import (
-	"testing"
-
-	"github.com/insolar/assured-ledger/ledger-core/configuration"
-	"github.com/insolar/assured-ledger/ledger-core/instrumentation/insapp"
-	"github.com/insolar/assured-ledger/ledger-core/server"
-)
-
-func Test_RunCloud(t *testing.T) {
```

Comment:
I wouldn't remove that because it's useful way to run cloud without tests and benches and do with that cloud what you want

Reply:
that is not the test, and it just starts cloud.

Reply:
we have cmd for that

Reply:
moved to testutils